### PR TITLE
Remove DQLite files when removing juju from a previously used machine

### PR DESCRIPTION
**Pull request by Aflynn50**
Juju allows for reusing of machines that have previously had juju on them but since been removed with `juju destroy-controller` or similar. When a machine is destroyed in this way, juju does not remove any of its files from the machine, instead, for safety, it leaves them there. They can be removed with a script juju installs:
```
/sbin/remove-juju-services
```
This script removes all juju files from the machine so that it can be bootstrapped from again. The problem is, it does not remove the dqlite files stored on disk.

This PR adds commands to remove the DQLite files too.
## Checklist

- [x] Code style: imports ordered, good names, simple structure, etc
- [x] Comments saying why design decisions were made
- [x] Go unit tests, with comments saying what you're testing

#### Add manual cloud
```
lxc launch ubuntu:22.04
```
Note down lxd container name and ip:
```
lxc list
```
Copy your ssh public key to the lxd container:
```
cat ~/.ssh/id_ed25519.pub | lxc exec <lxd-container-name> -- sh -c "cat >> /home/ubuntu/.ssh/authorized_keys"
```
Check you can ssh to it:
```
ssh ubuntu@<lxd-container-ip>
```
Exit ssh session and  bootstrap on it:
```
juju bootstrap manual/ubuntu@<lxd-container-ip>
```

#### Remove controller and clean up
Destroy controller
```
juju destroy-controller manual --no-prompt
```
Run juju provided cleanup script:
```
lxc exec <lxd-container-name> sudo /sbin/remove-juju-services
```
#### Retry bootstrap
Retry bootstrap, it should work as expected.
```
juju bootstrap manual/ubuntu@<lxd-container-ip>
```

## Links

**Launchpad bug:** https://bugs.launchpad.net/juju/+bug/2046186

**Jira card:** JUJU-5494

### CODE REVIEW

**Review by SimonRichardson — DISMISSED** [review]
Whilst you're here, can you remove the object store files as well?

```
/var/lib/juju/objectstore/*
```

Thanks.

File: scripts/removejujuservices.bash [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Copyright 2024 Canonical Ltd.
+# Licensed under the AGPLv3, see LICENCE file for details.
```
Comment (SimonRichardson) [inline scripts/removejujuservices.bash:4]: Do we need this?

**Review by SimonRichardson — APPROVED** [review]

File: scripts/removejujuservices.bash (hunk 1 above)
Comment (Aflynn50) [reply scripts/removejujuservices.bash:4]: Fair, I've changed my automated copyright thing to just do Go files

**Pull request merged**